tests/smoke-tests/test/smoke.test.ts

```diff
@@ -7,6 +7,12 @@ describe('Smoke Test', () => {
     ).resolves.toHaveProperty('status', 200);
   });
 
+  it('aicore client retrieves a list of deployments with custom destination', async () => {
```

Comment:
[q] Isn't one test for destination sufficient? Just wondering about the no. of calls.

Reply:
The story is that I want to show case two different usages for adding destination, and I created two functions in `sample-code`. I just want to use that function from somewhere. Unfortunately, there is no destination if run sample code locally. I can mock the destination, but this kind like not real. And calling ai-api isn't heavy, so I would rather leave it here. No strong opinion though.